code/__HELPERS/lists.dm

```diff
@@ -861,3 +861,7 @@
 // Pick something else from a list than we last picked
 /proc/pick_excluding(list/l, exclude)
 	return pick(l - exclude)
+
+///sort any value in a list
+/proc/sort_list(list/list_to_sort, cmp=/proc/cmp_text_asc)
```

Comment:
I think it's worth updating the comment (or the proc name) to show that this returns a sorted copy of the list, rather than sorting it in place.